packages/mongodb-memory-server-core/src/util/__tests__/dryBinary.test.ts

```diff
@@ -155,6 +157,46 @@ describe('DryBinary', () => {
     });
   });
 
+  describe('getBinaryName', () => {
+    afterEach(() => {
+      delete process.env[envName(ResolveConfigVariables.USE_ARCHIVE_NAME_FOR_BINARY_NAME)];
+    });
```

Comment:
this should probably be also run in a `beforeAll` to ensure it starts at default